agent/tool-scripts/prometheus-metrics

```diff
@@ -115,6 +114,11 @@ tool_stderr_file=$tool_output_dir/$tool-stderr.txt
 tool_log=$tool_output_dir/tool.log
 ## create directory to save the keys from openshift master, hosts
 mkdir -p /run/pbench/keys
+## check if the inventory variable is set
+if [[ -z "$inventory" ]] || [[ -s "$inventory" ]]; then
```

Comment:
The check is still wrong I thinK: -s checks for size greater than 0, in which case you don't want to complain. You should combine all the conditions into a single test too: `[[ -z "$inventory" || ! -s "$inventory" ]]`. But don't trust me: check that it is right! Or you might want to break the check up into if/elif/... cases and make each complaint more specific.